### ISSUE TITLE
How to get the details of unsegmented pixels of image

### ISSUE THREAD

**Issue by shubhamsinha47** [facebookresearch/detectron2#1711]
I am trying to build a segmentation model to identify the floor in any given image, based on my custom dataset which I am able to get.
![floor](https://user-images.githubusercontent.com/1926814/86420121-2ca05000-bcf3-11ea-8d86-12e668be5d22.png)
![image](https://user-images.githubusercontent.com/1926814/86420124-3033d700-bcf3-11ea-9a44-53ad630f23d9.png)

after going through the doc, I was able to mask out the floor, but I also want the unsegmented part as the non-floor area. 

below is the code. 

```
def get_cropped_floor(img, predictor, return_mapping=False):
    # convert to numpy
    img = np.array(img)[:, :, ::-1]
    img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)

    print('trying to get predication')
    # get prediction
    outputs = predictor(img)
    print('got the predicated value')

    # get boxes and masks
    ins = outputs["instances"]
    pred_masks = ins.get_fields()["pred_masks"]
    boxes = ins.get_fields()["pred_boxes"]

    # get main floor mask if the area is >= the mean area of boxes and is closes to the centre
    masker = pred_masks[np.argmin(
        [calculate_distance(x[0], x[1], int(img.shape[1] / 2), int(img.shape[0] / 2)) for i, x in
         enumerate(boxes.get_centers()) if (boxes[i].area() >= torch.mean(boxes.area()).to("cpu")).item()])].to(
        "cpu").numpy().astype(np.uint8)

    # mask image
    mask_out = cv2.bitwise_and(img, img, mask=masker)

    # find contours and boxes
    contours, hierarchy = cv2.findContours(masker.copy(), cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
    contour = contours[np.argmax([cv2.contourArea(x) for x in contours])]
    rotrect = cv2.minAreaRect(contour)
    box = cv2.boxPoints(rotrect)
    box = np.int0(box)

    if return_mapping:
        img = cv2.drawContours(img, [box], 0, (0, 0, 255), 10)
        img = cv2.drawContours(img, contours, -1, (255, 150,), 10)
        return mask_out, Image.fromarray(img)

    return mask_out
```

**Comment by ppwwyyxx** [comment]
From the description, it seems detectron2 has already done its job to provide the segmentation. The task of "getting unsegmented pixels", which I'm not sure exactly what it is, is not within the scope of detectron2 and can probably be done by users using the output from detectron2.
